### ISSUE TITLE
This is a new issue

### ISSUE THREAD

**Issue by kgarber** [kgarber/iFormAndroidMockup#2]
Will there be hard Rs everywhere?

### Teststst
- [ ] dasdasd
- [ ] asdasdasd

### Tasks that need to be done before removal
- [ ] Find out current usage
[insert file here]
- [ ] Given 90 day notice
[insert date and link here when announced]

### Tasks that need to be done before removal
- [ ] Find out current usage
[insert file here]
- [ ] Given 90 day notice
[insert date and link here when announced]
